www-client/otter-browser/otter_browser-0.9.12.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A lightweight QtWebKit browser"
+DESCRIPTION="Project aiming to recreate the best aspects of the \
+classic Opera (12.x) UI using Qt5."
+HOMEPAGE="http://otter-browser.org/"
+COPYRIGHT="2013-2017 Otter Browser Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/OtterBrowser/otter-browser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1a184561b9dbf3443cf1f83ad8604976b1b06cc00a93306331b364f237363e20"
+ADDITIONAL_FILES="otter.rdef.in"
+SOURCE_DIR="otter-browser-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	otter_browser$secondaryArchSuffix = $portVersion
+	app:OtterBrowser$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	hunspell$secondaryArchSuffix
+	qt5$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake$secondaryArchSuffix
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	hunspell${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel
+	devel:libssl$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
```

Comment:
@miqlas, when you copy these 2 lines to REQUIRES, don't forget to change the "devel:lib\*" into "lib:lib\*" to have:
```
	lib:libssl$secondaryArchSuffix
	lib:libcrypto$secondaryArchSuffix
```